src/styles/main.scss

```
@@ -1,3 +1,63 @@
 body {
   margin: 0;
 }
+$cell-size: 100px;
+$days:  "mon" 0, "tue" 1, "wed" 2, "thu" 3, "fri" 4, "sat" 5, "sun" 6;
+$gap-size: 1px;
+
+.calendar {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+  width: 8 * $cell-size;
+  padding: 20px;
+  gap: $gap-size;
+  margin: 0 auto;
+
+  &__day {
+    display: flex;
+    position: relative;
+    box-sizing: border-box;
+    justify-content: center;
+    align-items: center;
+    border: 1px solid black;
+    background-color: #eee;
+    height: $cell-size;
+    width: $cell-size;
+  }
+
+  &__day:hover {
+    cursor: pointer;
+    background-color: pink;
+    transform: translateY(-20px);
+    transition: all 0.5s;
```

Create variable for transitionDuration
also, don't use transition: all - it's to hard for the browser
Just use like that
```suggestion
    transition: transform $transitionDuration;
```

---

ок